event-sourcing/event-store-cassandra/src/main/scala/org/apache/james/eventsourcing/eventstore/cassandra/EventStoreDao.scala

```diff
@@ -51,10 +51,15 @@ class EventStoreDao @Inject() (val session: Session, val jsonEventSerializer: Js
   }
 
   private[cassandra] def appendAll(events: Iterable[Event]): SMono[Boolean] = {
-    val batch: BatchStatement = new BatchStatement
-    events.foreach((event: Event) => batch.add(insertEvent(event)))
-    SMono(cassandraAsyncExecutor.executeReturnApplied(batch))
-      .map(_.booleanValue())
+    if (events.size == 1) {
```

Comment:
> Glowroot instrumentation showed adding an event to an aggregate took 40ms

Is it related to batching or to LWT? What does glowroot show after this modification?

Reply:
> Is it related to batching or to LWT?

Well the stats with LWT inserts are generally within the few ms range generally in average 40ms.

> What does glowroot show after this modification?

Let me see.

Reply:
Well new measurement I did show 9 ms instead of 54 ms